modules/cluster-logging-elasticsearch-retention.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-elasticsearch.adoc
+
+[id="cluster-logging-elasticsearch-retention_{context}"]
+= Configuring Elasticsearch retention policy
+
+
+Each Elasticsearch index adds load and overhead to the Elasticsearch cluster. 
```

Comment:
@sichvoge I have reservations about this entire section.  I think its too much detail and it describes a resource a user can not edit without moving to unmanaged.   This seems like we may want a technical document someplace but do we really want it here?

Reply:
@jcantrill @sichvoge Is the retention policy only for how long to keep the rolled-over indices before deletion (as the name suggests!) using the cluster logging CR. Tthe user cannot change the rollover schedule without going unmanaged? 
In this paragraph, I wanted to describe the rollover process for folks who might not know the process and show the additions to the Elasticsearch CR made by by Cluster Logging CR.

Reply:
See: https://github.com/openshift/openshift-docs/pull/22403#discussion_r437790865